src/Infrastructure/Controller/Admin/OrganizationCrudController.php

```diff
@@ -34,6 +34,9 @@ public function configureFields(string $pageName): iterable
                 ->setLabel('Date de création')
                 ->setDisabled($pageName === Crud::PAGE_EDIT),
             TextField::new('siret')->setLabel('Siret'),
+            TextField::new('uuid')
+                ->setLabel('Id de l\'organisation')
```

Comment:
Souvent on parle même entre nous "d'UUID" donc peut-être qu'il vaut mieux dire ça aussi

Par ex si on dit à Mathieu F "quel est l'UUID de l'orga toto" :)

```suggestion
                ->setLabel('UUID de l\'organisation')
```